lib/rust/ast/generation/src/ast.rs

```diff
@@ -249,8 +273,8 @@ impl From<&str> for Type {
 impl From<&syn::Type> for Type {
     fn from(typ:&syn::Type) -> Self {
         let mut name = Name::default();
-        let mut path = Vec::default();
-        let mut args = Vec::default();
+        let mut path = Vec ::default();
+        let mut args = Vec ::default();
```

Comment:
Don't put a space after `Vec`, but you also shouldn't be specifying what you're defaulting.

Reply:
It doesn't typecheck otherwise.